lib/services/redis.js

```diff
@@ -47,18 +47,7 @@ function Redis (kuzzle, options, config) {
             return reject(error);
           }
 
-          // do not flush publicCache
-          if (this.settings.service === 'memoryStorage') {
-            return resolve(this);
-          }
-
-          this.client.flushdb(err => {
-            if (err) {
-              return reject(err);
-            }
-
-            resolve(this);
-          });
+          return resolve(this);
```

Comment:
nitpicking: Would be nice to have notice log in the 'ready' event to catch the case where a network configuration may force the client to reconnect repeatedly, like we've seen lately (i.e. client timeout set in haproxy) .